### DISCUSSION TITLE
Vercel (And others) Deployment

### DISCUSSION THREAD

**Discussion by HeroLink6** [NebulaServices/Nebula#216] [Ideas]
I thought it would be cool for the proxy to be deployed on Vercel or Netlify, because a lot of schools use those, so it wouldn't be blocked.

**Comment by PrimeKidMN** [comment]
It can, you have to down grade to the version from about a year ago though. I was able to deploy it through vercel with vue. Another thing is you can use chat gpt to help you update it to get it to deploy.

**Reply by iamawrench** [reply]
ok I'm dumb but how do you get it deployed on vercel because its notblocked

**Reply by PrimeKidMN** [reply]
I cant test it out right now because vercel is blocked but nebula can use koyeb. Deploy it as docker not a container and then I think you need to run it privileged but if it fails its probably that so just re do it without privilege. Also koyeb will cover you a free domain so there you go.

**Reply by PrimeKidMN** [reply]
Oh yeah follow docker-compose instructions about setting up the .toml files, you can use cli or manual editing but cli is probably gonna be easier before you deploy to koyeb.
